README.md

```diff
@@ -351,6 +351,9 @@ Retool also has a health check endpoint that you can set up to monitor livelines
   - If you have not enabled SSL yet, you will need to add the line `COOKIE_INSECURE=true` to your `docker.env` file / environment configuration so that the authentication cookies can be sent over http. Make sure to run `sudo docker-compose up -d` after modifying the `docker.env` file.
 - `TypeError: Cannot read property 'licenseVerification' of null` or `TypeError: Cannot read property 'name' of null`
   - There is an issue with your license key. Double check that the license key is correct and that it has no trailing whitespaces.
+- I want to use a private IP of the machine, not the default public one
+    - When you run `./install.sh`, instead of just clicking enter, type in your private IP. If you want to change this after it has already been set, modify the DOMAINS variable in the docker.env file.
```

Comment:
```suggestion
    - When you run `./install.sh`, instead of just clicking enter, type in your private IP. If you want to change this after it has already been set, modify the DOMAINS variable in the `docker.env` file.
```